site/content/en/docs/manual/advanced/organization.md

```diff
@@ -5,57 +5,128 @@ weight: 2
+| **Email**           | Specifies the email address of the user who is being added to the **Organization**.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
+| Role drop-down list | Defines the role of the user which sets the level of access within the **Organization**: <br><li>**Worker**: Has access only to the tasks, projects, and jobs assigned to them. <li>**Supervisor**: Can create and assign jobs, tasks, and projects to the **Organization** members. <li>**Maintainer**: Has the same capabilities as the **Supervisor**, but with additional visibility over all tasks and projects created by other members, complete access to **Cloud Storages**, and the ability to modify members and their roles. <li>**Owner**: role assigned to the creator of the organization by default. Has maximum capabilities and cannot be changed or assigned to the other user. |
+| **Invite more**     | Button to add another user to the **Organization**.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
+
+<!--lint enable maximum-line-length-->
+
+Members of **Organization** will appear on the **Organization page**.
 
 ![](/images/image237.jpg)
 
-The member can leave the organization on his own by clicking `Leave organization` on the organization settings page.
+The member of the organization can leave the organization
```

Comment:
Let's also drop a line how organization maintainer can remove members from the organization